patches/@bitauth+libauth+1.19.1.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/node_modules/@bitauth/libauth/build/main/lib/format/log.js b/node_modules/@bitauth/libauth/build/main/lib/format/log.js
+index 6a1f908..0fedcfc 100644
+--- a/node_modules/@bitauth/libauth/build/main/lib/format/log.js
```

Comment:
Since this patch is disabling functionality instead of patching it, please provide at least a comment

Reply:
Not sure whether it is possible to write comments in .patch files. A detailed explanation is in WAL-294.

Reply:
Thr point is that we need something to link it from the actual code. Your patch is literally commenting some code out, of course you then can add a comment in that commented out code, explaining why or linking to that ticket

Reply:
Done " --> this regex expression does not work in RN, see WAL-294 for details"